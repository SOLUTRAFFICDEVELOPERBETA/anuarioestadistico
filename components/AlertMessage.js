import React from 'react'
import { IconButton, makeStyles, Snackbar } from '@material-ui/core'
import { Close } from '@material-ui/icons'
import { green, red, orange } from '@material-ui/core/colors'
import AlertContext from '../contexts/alert'

const useStyles = makeStyles(theme => ({
  root: {
    position: 'fixed',
    top: 0,
    left: 0,
    padding: theme.spacing(1.5),
    width: '100%',
    zIndex: 1600,

    [theme.breakpoints.down('md')]: {
      width: 'auto',
      height: 'fit-content',
      bottom: theme.spacing(2),
      right: theme.spacing(3),
      borderRadius: theme.spacing(1),
      left: 'auto',
      margin: theme.spacing(1, 0)
    }
  },
  success: {
    backgroundColor: green[600],
    fontSize: '1.5rem',
    color: theme.palette.getContrastText(green[600])
  },
  error: {
    backgroundColor: red[600],
    fontSize: '1.5rem',
    color: theme.palette.getContrastText(red[600])
  },
  warning: {
    backgroundColor: orange[600],
    color: theme.palette.getContrastText(orange[600]),
    fontSize: '1.5rem'
  }
}))

const AlertMessage = () => {
  const classes = useStyles()
  const { message, closeMessage } = React.useContext(AlertContext)

  if (!message) return null

  const { text, category, time } = message

  return (
    <Snackbar
      open={Boolean(message)}
      autoHideDuration={time}
      onClose={closeMessage}
      anchorOrigin={{
        vertical: 'bottom',
        horizontal: 'center'
      }}
      ContentProps={{
        classes: {
          root: classes[category]
        }
      }}
      message={text}
      action={
        <React.Fragment>
          <IconButton size="small" color="inherit" onClick={closeMessage}>
            <Close fontSize="small" />
          </IconButton>
        </React.Fragment>
      }
    />
  )
}

export default AlertMessage
