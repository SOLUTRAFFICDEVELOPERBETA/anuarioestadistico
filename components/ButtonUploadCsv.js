import React from 'react';
import PropTypes from 'prop-types';
import { Box, Button } from '@material-ui/core';
import { CloudUpload } from '@material-ui/icons';

/**
 *
 * @param {title} string titulo del componente,
 * @param {handleFilesAnnex} fn función para subir el archivo,
 * @param {name} string Nombre deñ archivo,
 * @param {id} string identifdicador  del campo,
 * @param {htmlFor} string Identificador del Label,
 */
const ButtonUploadCsv = ({ title, handleFilesAnnex, name, id, htmlFor }) => {
    return (
        <Box>
            <input
                type={'file'}
                name={name}
                id={id}
                multiple={false}
                style={{ display: 'none' }}
                onChange={(e) => handleFilesAnnex(e.target.files)}
            />
            <label htmlFor={htmlFor}>
                <Button
                    endIcon={<CloudUpload />}
                    component={'div'}
                    size={'small'}
                    fullWidth={true}
                    variant={'contained'}
                    color={'secondary'}>
                    {title}
                </Button>
            </label>
        </Box>
    );
};
ButtonUploadCsv.propTypes = {
    title: PropTypes.string,
    handleFilesAnnex: PropTypes.func,
    name: PropTypes.string,
    id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
    htmlFor: PropTypes.string
};
export default ButtonUploadCsv;
