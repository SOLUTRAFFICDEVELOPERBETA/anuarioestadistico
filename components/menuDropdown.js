/* eslint-disable react/jsx-key */
import React from 'react';
import PropTypes from 'prop-types';
import { AppBar, Tabs, Tab, Box, useTheme } from '@material-ui/core';
import { css } from '@emotion/core';

/**
 * Componente que permite visualizar cada tab.
 * @param {children: any, value: string, index: any} props del componente.
 */
function TabPanel(props) {
    const { children, value, index, ...other } = props;
    return (
        <div
            role="tabpanel"
            hidden={value !== index}
            id={`scrollable-auto-tabpanel-${index}`}
            aria-labelledby={`scrollable-auto-tab-${index}`}
            {...other}>
            {value === index && (
                <Box
                    css={css`
                        padding: 3rem;

                        @media (max-width: 768px) {
                            padding: 0rem;
                        }
                    `}>
                    <Box>{children}</Box>
                </Box>
            )}
        </div>
    );
}

TabPanel.propTypes = {
    children: PropTypes.node,
    index: PropTypes.any.isRequired,
    value: PropTypes.any.isRequired
};

// MÃ©todo para cambiar de panel.
function a11yProps(index) {
    return {
        id: `scrollable-auto-tab-${index}`,
        'aria-controls': `scrollable-auto-tabpanel-${index}`
    };
}
/**
 * Componente que permite visualizar los panel de la plataforma.
 * @param {data: {}} props del componente.
 */
const MenuTabsMain = ({ data }) => {
    const [value, setValue] = React.useState(0);

    const theme = useTheme();
    const handleChange = (event, newValue) => {
        setValue(newValue);
    };

    return (
        <div
            css={css`
                padding-left: 1rem;
                padding-right: 1rem;
                flex-flow: 1;
                width: 100%;
                background-color: ${theme.palette.background.paper};

                @media (max-width: 768px) {
                    padding-left: 0rem;
                    padding-right: 0rem;
                }
            `}>
            <AppBar position="static" color="secondary">
                <Tabs
                    value={value}
                    onChange={handleChange}
                    indicatorColor="primary"
                    textColor="inherit"
                    variant="scrollable"
                    scrollButtons="auto"
                    aria-label="scrollable auto tabs example">
                    {data.map((value, index) => (
                        <Tab label={value.label} {...a11yProps(index)} />
                    ))}
                </Tabs>
            </AppBar>
            {data.map((item, index) => {
                const { Component } = item;
                return (
                    <TabPanel value={value} index={index}>
                        <div>
                            <Component />
                        </div>
                    </TabPanel>
                );
            })}
        </div>
    );
};
MenuTabsMain.propTypes = {
    data: PropTypes.array.isRequired
};

export { MenuTabsMain };
