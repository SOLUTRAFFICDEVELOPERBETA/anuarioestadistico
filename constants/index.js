


// Módulos
import TerrestrialMobility from "../Layout/Anuario/MovilidadTerrestre"
import HistoricosGeograficos from "../Layout/Anuario/historicosGeograficos"
import Demografia from "../Layout/Anuario/demografia"
import Salud from "../Layout/Anuario/salud"
import Educacion from "../Layout/Anuario/educacion"
import Reacreacion from "../Layout/Anuario/recreacion"
import ServiciosPublicos from "../Layout/Anuario/serviciosPublicos"
import SeguridadConvivencia from "../Layout/Anuario/SeguridadConvivencia"
import ViolenciaIntrafamiliar from "../Layout/Anuario/violenciaIntrafamiliar"
import HaciendaPublica from "../Layout/Anuario/haciendaPublica"
import SectorFinanciero from "../Layout/Anuario/sectorFinanciero"
import SectorAgropecuario from "../Layout/Anuario/sectorAgropecuario"
import ConstruccionVivienda from "../Layout/Anuario/contruccionVivienda"
import SectorEmpresarial from "../Layout/Anuario/sectorEmpresarial"
import ComercioExterior from "../Layout/Anuario/comercioExterior"
import MovimientoAereo from "../Layout/Anuario/movimientoAereo"

export const PROFILE_IMG = 'https://firebasestorage.googleapis.com/v0/b/wise-365ab.appspot.com/o/assets%2Fprofile_default.jpg?alt=media&token=9779b4ee-2ee5-49c1-b24b-720e5854867b'
export const AUTH_ITEM = 'WISE_AUTH'

// Fecha
export const INPUT_DATE = 'YYYY-MM-DD'


export const COLOR_GRAPH = {
    backgroundColor: ['#629749', '#ffb04c', '#ff833a', '#f05545'],
    borderColor: ['#003d00', '#bc5100', '#ac1900', '#7f0000'],
    borderWidth: 1,
    hoverBackgroundColor: ['#33691e', '#f57f17', '#e65100', '#b71c1c'],
    hoverBorderColor: ['#003d00', '#bc5100', '#ac1900', '#7f0000'],
    datalabels: {
        color: ['#629749', '#ffb04c', '#ff833a', '#f05545'],
        backgroundColor: '#fff',
        borderColor: ['#003d00', '#bc5100', '#ac1900', '#7f0000']
    },
}

export const LinksMenu = [
    {
        label: 'Movilidad Terrestre', href: '#!',
        Component: function () {
            return <TerrestrialMobility />
        }
    },
    {
        label: 'Aspectos  Históricos  y  Geográficos', href: '#!',
        Component: function () {
            return <HistoricosGeograficos />
        }
    },
    {
        label: 'Demografía', href: '#!',
        Component: function () {
            return <Demografia />
        }
    },
    {
        label: 'Salud', href: '#!',
        Component: function () {
            return <Salud />
        }
    },
    {
        label: 'Educación', href: '#!',
        Component: function () {
            return <Educacion />
        }
    },
    {
        label: 'Recreación y zonas verdes', href: '#!',
        Component: function () {
            return <Reacreacion />
        }
    },
    {
        label: 'Servicios Públicos', href: '#!',
        Component: function () {
            return <ServiciosPublicos />
        }
    },
    {
        label: 'Seguridad y convivencia', href: '#!',
        Component: function () {
            return <SeguridadConvivencia />
        }
    },
    {
        label: 'Violencia Intrafamiliar', href: '#!',
        Component: function () {
            return <ViolenciaIntrafamiliar />
        }
    },
    {
        label: 'Hacienda Públicas', href: '#!',
        Component: function () {
            return <HaciendaPublica />
        }
    },
    {
        label: 'Sector Financiero', href: '#!',
        Component: function () {
            return <SectorFinanciero />
        }
    },
    {
        label: 'Sector Agropecuario', href: '#!',
        Component: function () {
            return <SectorAgropecuario />
        }
    },
    {
        label: 'Construcción y vivienda', href: '#!',
        Component: function () {
            return <ConstruccionVivienda />
        }
    },
    {
        label: 'Sector Empresarial', href: '#!',
        Component: function () {
            return <SectorEmpresarial />
        }
    },
    {
        label: 'Comercio Exterior', href: '#!',
        Component: function () {
            return <ComercioExterior />
        }
    },
    {
        label: 'Movimiento Aéreo', href: '#!',
        Component: function () {
            return <MovimientoAereo />
        }
    },


]


