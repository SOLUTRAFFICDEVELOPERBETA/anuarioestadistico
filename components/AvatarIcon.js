/* eslint-disable indent */
import React from 'react'
import { Avatar, makeStyles, Popover, List, ListItem, ListItemText, Button, ListItemIcon } from '@material-ui/core'
import { AccountBoxTwoTone, ExitToApp } from '@material-ui/icons'
import { PROFILE_IMG } from '../constants'
import moment from 'moment'
import { useRouter } from 'next/router'
import AuthContext from '../contexts/auth'

const useStyles = makeStyles(theme => ({
  avatar: {
    width: theme.spacing(3),
    height: theme.spacing(3),
    color: theme.palette.primary.contrastText,
    backgroundColor: theme.palette.primary.light
  },
  button: {
    color: theme.palette.primary.main,
    backgroundColor: theme.palette.primary.contrastText,
    '&:hover': {
      backgroundColor: theme.palette.primary.contrastText
    }
  }
}))

const AvatarIcon = () => {
  const router = useRouter()
  const classes = useStyles()
  const [anchorEl, setAnchorEl] = React.useState(null)
  const { user, onLogOut } = React.useContext(AuthContext)

  const handleClick = (event) => {
    setAnchorEl(event.currentTarget)
  }

  const handleClose = () => {
    setAnchorEl(null)
  }

  const auth = Boolean(user)
  const open = Boolean(anchorEl)
  const id = open ? 'simple-popover' : undefined

  return (
    <React.Fragment>
      {auth ? (
        <Button
          color="primary"
          className={classes.button}
          variant="contained"
          onClick={handleClick}
          endIcon={
            <Avatar className={classes.avatar} alt="img-user" src={user && user.avatar ? user.avatar.url : PROFILE_IMG} />
          }
        >
          {moment().format('ll')}
        </Button>
      ) : (
          <Button
            color="inherit"
            variant="outlined"
            size="small"
            onClick={() => router.push('/auth')}
          >
            iniciar sesión
          </Button>
        )}
      {open && (
        <Popover
          id={id}
          open={open}
          anchorEl={anchorEl}
          onClose={handleClose}
          anchorOrigin={{
            vertical: 'bottom',
            horizontal: 'right'
          }}
          transformOrigin={{
            vertical: 'top',
            horizontal: 'right'
          }}
        >
          <List>
            <ListItem dense button onClick={() => router.push('/')}>
              <ListItemIcon>
                <AccountBoxTwoTone color="primary" />
              </ListItemIcon>
              <ListItemText primary="Perfil" />
            </ListItem>
            <ListItem dense button onClick={onLogOut}>
              <ListItemIcon>
                <ExitToApp color="primary" />
              </ListItemIcon>
              <ListItemText primary="Cerrar Sesión" />
            </ListItem>
          </List>
        </Popover>
      )}
    </React.Fragment>
  )
}

export default AvatarIcon
