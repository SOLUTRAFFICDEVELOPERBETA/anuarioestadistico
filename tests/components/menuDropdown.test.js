import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import React from 'react';
import { MenuTabsMain } from '../../components/menuDropdown';
import {LinksMenu} from '../../constants'

describe('Pruebas en <MenuTabsMain/>', () => {
    const wrapper = shallow(

        <MenuTabsMain
            data={LinksMenu}
        />

    )

    test('Debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('debe retornar los componentes de la rutas', () => {
        expect(wrapper.find('Component').exists()).toBe(true)
       
    });

    test('debe de  devolver los 16 módulos del anuario estadístico', ()=> {
        expect(wrapper.find('TabPanel').at(15).exists()).toBe(true);
    })

   
})