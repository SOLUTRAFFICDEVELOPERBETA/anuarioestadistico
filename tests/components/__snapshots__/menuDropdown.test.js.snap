// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <MenuTabsMain/> Debe mostrarse correctamente 1`] = `
<EmotionCssPropInternal
  __EMOTION_LABEL_PLEASE_DO_NOT_USE__="MenuTabsMain"
  __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
  css={
    Object {
      "map": "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIm1lbnVEcm9wZG93bi5qcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFrRW9CIiwiZmlsZSI6Im1lbnVEcm9wZG93bi5qcyIsInNvdXJjZXNDb250ZW50IjpbIi8qIGVzbGludC1kaXNhYmxlIHJlYWN0L2pzeC1rZXkgKi9cbmltcG9ydCBSZWFjdCBmcm9tICdyZWFjdCc7XG5pbXBvcnQgUHJvcFR5cGVzIGZyb20gJ3Byb3AtdHlwZXMnO1xuaW1wb3J0IHsgQXBwQmFyLCBUYWJzLCBUYWIsIEJveCwgdXNlVGhlbWUgfSBmcm9tICdAbWF0ZXJpYWwtdWkvY29yZSc7XG5pbXBvcnQgeyBjc3MgfSBmcm9tICdAZW1vdGlvbi9jb3JlJztcblxuLyoqXG4gKiBAZGVzY3JpcHRpb24gQ29tcG9uZW50ZSBxdWUgcGVybWl0ZSB2aXN1YWxpemFyIGNhZGEgdGFiLlxuICogQHBhcmFtIHthbnl9IGNoaWxkcmVuXG4gKiBAcGFyYW0ge1N0cmluZ30gdmFsdWVcbiAqIEBwYXJhbSB7YW55fSBpbmRleFxuICogQ29uc3VsdGUgbG9zIGVsZW1lbnRvcyB1dGlsaXphZG9zIGVuIHtAbGluayBodHRwczovL21hdGVyaWFsLXVpLmNvbS8gfCBNYXRlcmlhbC11aX1cbiAqL1xuZnVuY3Rpb24gVGFiUGFuZWwocHJvcHMpIHtcbiAgICBjb25zdCB7IGNoaWxkcmVuLCB2YWx1ZSwgaW5kZXgsIC4uLm90aGVyIH0gPSBwcm9wcztcbiAgICByZXR1cm4gKFxuICAgICAgICA8ZGl2XG4gICAgICAgICAgICByb2xlPVwidGFicGFuZWxcIlxuICAgICAgICAgICAgaGlkZGVuPXt2YWx1ZSAhPT0gaW5kZXh9XG4gICAgICAgICAgICBpZD17YHNjcm9sbGFibGUtYXV0by10YWJwYW5lbC0ke2luZGV4fWB9XG4gICAgICAgICAgICBhcmlhLWxhYmVsbGVkYnk9e2BzY3JvbGxhYmxlLWF1dG8tdGFiLSR7aW5kZXh9YH1cbiAgICAgICAgICAgIHsuLi5vdGhlcn0+XG4gICAgICAgICAgICB7dmFsdWUgPT09IGluZGV4ICYmIChcbiAgICAgICAgICAgICAgICA8Qm94XG4gICAgICAgICAgICAgICAgICAgIGNzcz17Y3NzYFxuICAgICAgICAgICAgICAgICAgICAgICAgcGFkZGluZzogM3JlbTtcblxuICAgICAgICAgICAgICAgICAgICAgICAgQG1lZGlhIChtYXgtd2lkdGg6IDc2OHB4KSB7XG4gICAgICAgICAgICAgICAgICAgICAgICAgICAgcGFkZGluZzogMHJlbTtcbiAgICAgICAgICAgICAgICAgICAgICAgIH1cbiAgICAgICAgICAgICAgICAgICAgYH0+XG4gICAgICAgICAgICAgICAgICAgIDxCb3g+e2NoaWxkcmVufTwvQm94PlxuICAgICAgICAgICAgICAgIDwvQm94PlxuICAgICAgICAgICAgKX1cbiAgICAgICAgPC9kaXY+XG4gICAgKTtcbn1cblxuVGFiUGFuZWwucHJvcFR5cGVzID0ge1xuICAgIGNoaWxkcmVuOiBQcm9wVHlwZXMubm9kZSxcbiAgICBpbmRleDogUHJvcFR5cGVzLmFueS5pc1JlcXVpcmVkLFxuICAgIHZhbHVlOiBQcm9wVHlwZXMuYW55LmlzUmVxdWlyZWRcbn07XG5cbi8vIE3DqXRvZG8gcGFyYSBjYW1iaWFyIGRlIHBhbmVsLlxuZnVuY3Rpb24gYTExeVByb3BzKGluZGV4KSB7XG4gICAgcmV0dXJuIHtcbiAgICAgICAgaWQ6IGBzY3JvbGxhYmxlLWF1dG8tdGFiLSR7aW5kZXh9YCxcbiAgICAgICAgJ2FyaWEtY29udHJvbHMnOiBgc2Nyb2xsYWJsZS1hdXRvLXRhYnBhbmVsLSR7aW5kZXh9YFxuICAgIH07XG59XG4vKipcbiAqIEBwYXJhbSBDb21wb25lbnRlIHF1ZSBwZXJtaXRlIHZpc3VhbGl6YXIgbG9zIHBhbmVsIGRlIGxhIHBsYXRhZm9ybWEuXG4gKiBAcGFyYW0ge29iamVjdCB7fX0gZGF0YSBkZWwgY29tcG9uZW50ZS5cbiAqIENvbnN1bHRlIGxvcyBlbGVtZW50b3MgdXRpbGl6YWRvcyBlbiB7QGxpbmsgaHR0cHM6Ly9tYXRlcmlhbC11aS5jb20vIHwgTWF0ZXJpYWwtdWl9XG4gKi9cbmNvbnN0IE1lbnVUYWJzTWFpbiA9ICh7IGRhdGEgfSkgPT4ge1xuICAgIGNvbnN0IFt2YWx1ZSwgc2V0VmFsdWVdID0gUmVhY3QudXNlU3RhdGUoMCk7XG5cbiAgICBjb25zdCB0aGVtZSA9IHVzZVRoZW1lKCk7XG4gICAgY29uc3QgaGFuZGxlQ2hhbmdlID0gKGV2ZW50LCBuZXdWYWx1ZSkgPT4ge1xuICAgICAgICBzZXRWYWx1ZShuZXdWYWx1ZSk7XG4gICAgfTtcblxuICAgIHJldHVybiAoXG4gICAgICAgIDxkaXZcbiAgICAgICAgICAgIGNzcz17Y3NzYFxuICAgICAgICAgICAgICAgIHBhZGRpbmctbGVmdDogMXJlbTtcbiAgICAgICAgICAgICAgICBwYWRkaW5nLXJpZ2h0OiAxcmVtO1xuICAgICAgICAgICAgICAgIGZsZXgtZmxvdzogMTtcbiAgICAgICAgICAgICAgICB3aWR0aDogMTAwJTtcbiAgICAgICAgICAgICAgICBiYWNrZ3JvdW5kLWNvbG9yOiAke3RoZW1lLnBhbGV0dGUuYmFja2dyb3VuZC5wYXBlcn07XG5cbiAgICAgICAgICAgICAgICBAbWVkaWEgKG1heC13aWR0aDogNzY4cHgpIHtcbiAgICAgICAgICAgICAgICAgICAgcGFkZGluZy1sZWZ0OiAwcmVtO1xuICAgICAgICAgICAgICAgICAgICBwYWRkaW5nLXJpZ2h0OiAwcmVtO1xuICAgICAgICAgICAgICAgIH1cbiAgICAgICAgICAgIGB9PlxuICAgICAgICAgICAgPEFwcEJhciBwb3NpdGlvbj1cInN0YXRpY1wiIGNvbG9yPVwic2Vjb25kYXJ5XCI+XG4gICAgICAgICAgICAgICAgPFRhYnNcbiAgICAgICAgICAgICAgICAgICAgdmFsdWU9e3ZhbHVlfVxuICAgICAgICAgICAgICAgICAgICBvbkNoYW5nZT17aGFuZGxlQ2hhbmdlfVxuICAgICAgICAgICAgICAgICAgICBpbmRpY2F0b3JDb2xvcj1cInByaW1hcnlcIlxuICAgICAgICAgICAgICAgICAgICB0ZXh0Q29sb3I9XCJpbmhlcml0XCJcbiAgICAgICAgICAgICAgICAgICAgdmFyaWFudD1cInNjcm9sbGFibGVcIlxuICAgICAgICAgICAgICAgICAgICBzY3JvbGxCdXR0b25zPVwiYXV0b1wiXG4gICAgICAgICAgICAgICAgICAgIGFyaWEtbGFiZWw9XCJzY3JvbGxhYmxlIGF1dG8gdGFicyBleGFtcGxlXCI+XG4gICAgICAgICAgICAgICAgICAgIHtkYXRhLm1hcCgodmFsdWUsIGluZGV4KSA9PiAoXG4gICAgICAgICAgICAgICAgICAgICAgICA8VGFiIGtleT17aW5kZXh9IGxhYmVsPXt2YWx1ZS5sYWJlbH0gey4uLmExMXlQcm9wcyhpbmRleCl9IC8+XG4gICAgICAgICAgICAgICAgICAgICkpfVxuICAgICAgICAgICAgICAgIDwvVGFicz5cbiAgICAgICAgICAgIDwvQXBwQmFyPlxuICAgICAgICAgICAge2RhdGEubWFwKChpdGVtLCBpbmRleCkgPT4ge1xuICAgICAgICAgICAgICAgIGNvbnN0IHsgQ29tcG9uZW50IH0gPSBpdGVtO1xuICAgICAgICAgICAgICAgIHJldHVybiAoXG4gICAgICAgICAgICAgICAgICAgIDxUYWJQYW5lbCBrZXk9e2luZGV4fSB2YWx1ZT17dmFsdWV9IGluZGV4PXtpbmRleH0+XG4gICAgICAgICAgICAgICAgICAgICAgICA8ZGl2PlxuICAgICAgICAgICAgICAgICAgICAgICAgICAgIDxDb21wb25lbnQgLz5cbiAgICAgICAgICAgICAgICAgICAgICAgIDwvZGl2PlxuICAgICAgICAgICAgICAgICAgICA8L1RhYlBhbmVsPlxuICAgICAgICAgICAgICAgICk7XG4gICAgICAgICAgICB9KX1cbiAgICAgICAgPC9kaXY+XG4gICAgKTtcbn07XG5NZW51VGFic01haW4ucHJvcFR5cGVzID0ge1xuICAgIGRhdGE6IFByb3BUeXBlcy5hcnJheS5pc1JlcXVpcmVkXG59O1xuXG5leHBvcnQgeyBNZW51VGFic01haW4gfTtcbiJdfQ== */",
      "name": "h605xt-MenuTabsMain-MenuTabsMain",
      "next": undefined,
      "styles": "padding-left:1rem;padding-right:1rem;flex-flow:1;width:100%;background-color:#fff;@media (max-width:768px){padding-left:0rem;padding-right:0rem;};label:MenuTabsMain;;label:MenuTabsMain;",
      "toString": [Function],
    }
  }
>
  <WithStyles(ForwardRef(AppBar))
    color="secondary"
    position="static"
  >
    <WithStyles(ForwardRef(Tabs))
      aria-label="scrollable auto tabs example"
      indicatorColor="primary"
      onChange={[Function]}
      scrollButtons="auto"
      textColor="inherit"
      value={0}
      variant="scrollable"
    >
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-0"
        id="scrollable-auto-tab-0"
        key="0"
        label="Movilidad Terrestre"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-1"
        id="scrollable-auto-tab-1"
        key="1"
        label="Aspectos  Históricos  y  Geográficos"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-2"
        id="scrollable-auto-tab-2"
        key="2"
        label="Demografía"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-3"
        id="scrollable-auto-tab-3"
        key="3"
        label="Salud"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-4"
        id="scrollable-auto-tab-4"
        key="4"
        label="Educación"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-5"
        id="scrollable-auto-tab-5"
        key="5"
        label="Recreación y zonas verdes"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-6"
        id="scrollable-auto-tab-6"
        key="6"
        label="Servicios Público"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-7"
        id="scrollable-auto-tab-7"
        key="7"
        label="Seguridad y convivencia"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-8"
        id="scrollable-auto-tab-8"
        key="8"
        label="Violencia Intrafamiliar"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-9"
        id="scrollable-auto-tab-9"
        key="9"
        label="Hacienda Pública"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-10"
        id="scrollable-auto-tab-10"
        key="10"
        label="Sector Financiero"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-11"
        id="scrollable-auto-tab-11"
        key="11"
        label="Sector Agropecuario"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-12"
        id="scrollable-auto-tab-12"
        key="12"
        label="Construcción y vivienda"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-13"
        id="scrollable-auto-tab-13"
        key="13"
        label="Sector Empresarial"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-14"
        id="scrollable-auto-tab-14"
        key="14"
        label="Comercio Exterior"
      />
      <WithStyles(ForwardRef(Tab))
        aria-controls="scrollable-auto-tabpanel-15"
        id="scrollable-auto-tab-15"
        key="15"
        label="Movimiento Aéreo"
      />
    </WithStyles(ForwardRef(Tabs))>
  </WithStyles(ForwardRef(AppBar))>
  <TabPanel
    index={0}
    key="0"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={1}
    key="1"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={2}
    key="2"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={3}
    key="3"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={4}
    key="4"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={5}
    key="5"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={6}
    key="6"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={7}
    key="7"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={8}
    key="8"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={9}
    key="9"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={10}
    key="10"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={11}
    key="11"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={12}
    key="12"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={13}
    key="13"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={14}
    key="14"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
  <TabPanel
    index={15}
    key="15"
    value={0}
  >
    <div>
      <Component />
    </div>
  </TabPanel>
</EmotionCssPropInternal>
`;
